spec/cases/points_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'spec_helper'
+
+describe DummyMetrics do
+  before do
+    stub_request(:get, "http://localhost:8086/query")
+      .with(
+        query: { q: 'select * from "dummy"', u: "root", p: "root", precision: 's', db: 'db' }
+      )
+      .to_return(body: File.read('./spec/fixtures/single_series.json'))
```

Comment:
Style/DotPosition: Place the . on the previous line, together with the method call receiver.